{"name": "test_login_0_failed[data0]", "status": "passed", "attachments": [{"name": "log", "source": "8c95667d-2533-49eb-8d28-1dc19ea2a14c-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "data", "value": "{'phone': '13443256789', 'password': '12', 'expected': '此账号没有经过授权，请联系管理员!'}"}], "start": 1585810350674, "stop": 1585810352662, "uuid": "435e05ca-409e-4ec9-a1cd-502b248a6e1e", "historyId": "ca9e3378364151b0613ff6eeca8627e0", "testCaseId": "c45059a0b3047499911997903855e071", "fullName": "test_login.TestLogin#test_login_0_failed", "labels": [{"name": "tag", "value": "all"}, {"name": "tag", "value": "login"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-D7A50AL"}, {"name": "thread", "value": "11484-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_login"}]}